datanode/src/storage_api.py

```diff
@@ -91,7 +91,8 @@
 # VERSION = 'TCL4.0.2.008'  # sync with master branch for multi-grid and docker updates
 # VERSION = 'TCL4.0.3.009'  # Added support for UVRs
 # VERSION = 'TCL4.0.3.010'  # Fixed backwards compatibility issue
-VERSION = 'TCL4.1.0.011'  # Updated UVR schema (not API-compatible with previous UVR schema)
+# VERSION = 'TCL4.1.0.011'  # Updated UVR schema (not API-compatible with previous UVR schema)
+VERSION = 'TCL4.1.1.012' # Improved validation and error messages
```

Comment:
For versions, I would have expected build .011 to have changed the minor version number, and .012 to not change the minor version number. No change required since this version would be the same, but make sure the swaggerhub version number matches (without the build #). 

There are instructions up top: "MINOR changes when there is a non-breaking API change (new fields or methods)".

Reply:
We need to define exactly what "breaking the API" means (#56).  In this case, the API will return UVRs that are incompatible with UVRs from the previous version.  A client designed to validate the old UVR format will be broken by this API change.